project_scrum/models/project_scrum_meeting.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# Copyright <2017> <Tenovar Ltd>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models, fields, api, _
+
+
+class ProjectScrumMeeting(models.Model):
+    _name = 'project.scrum.meeting'
+    _description = 'Project Scrum Daily Meetings'
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+
+    project_id = fields.Many2one(
+        comodel_name='project.project',
+        string='Project',
+        ondelete='cascade',
+        index=True,
+        track_visibility='onchange',
+        change_default=True,
+        required=True,
+    )
+    sprint_id = fields.Many2one(
+        comodel_name='project.scrum.sprint',
+        string='Sprint',
+    )
+    date_meeting = fields.Date(
+        string='Date',
+        required=True,
+    )
+    user_id_meeting = fields.Many2one(
```

Comment:
So these are "meetings" where only one person can be present?

Reply:
@NL66278 Going through the rest of the code and the templates I see that this is supposed to be the organizer of the meeting. I might be wrong though, you might want to check the mail_template.xml. And if this field is turned to many2many how will the template change?